app_web/src/logic/uimodel.js

```diff
@@ -26,7 +26,8 @@ class UIModel
             map( value => ({mainFile: value, additionalFiles: undefined})),
         );
         this.flavour = app.flavourChanged$.pipe(pluck("event", "msg")); // TODO gltfModelPathProvider needs to be changed to accept flavours explicitely
-        this.scene = app.sceneChanged$.pipe(pluck("event", "msg"));
+        const sceneSubject = new Subject();
+        this.scene = app.sceneChanged$.pipe(pluck("event", "msg"), multicast(sceneSubject));
```

Comment:
this probably is not necessary, as there are no side effects to the the sceneChanged observable

Reply:
removed multicast